apps/server/src/modules/files/uc/sync-embedded-files.uc.ts

```diff
@@ -82,6 +91,14 @@ export class SyncEmbeddedFilesUc {
 			}
 		}
 
+		if (entity instanceof Submission && entity.comment !== undefined) {
```

Comment:
```suggestion
		if (entity instanceof Submission && entity.comment) {
```
Ich glaube so wird auch reichen?